packages/fxa-payments-server/src/components/PaymentConfirmation/index.test.tsx

```diff
@@ -229,6 +261,80 @@ describe('PaymentConfirmation', () => {
     expect(queryByText(defaultButtonLabel)).not.toBeInTheDocument();
   });
 
+  //This one with config
```

Comment:
nit: this comment doesn't seem necessary since the test description mentions "firestore config"

Reply:
Whoops, yup that was their for my brain to keep track of things.